src/main/java/com/company/logic/EmployeeLogic.java

```diff
@@ -10,15 +13,31 @@
  */
 public class EmployeeLogic {
 
-    private static final String GET_BY_ID = "Employee.getById";
-
     public Employee get(int idEmployee) {
-        Employee employee = null;
+        Employee employee = new Employee();
+        SqlSession session = null;
+        try {
+            session = new Connection().getSqlSession();
+            if (session != null) {
+                employee = session.selectOne(ConstantData.GET_BY_ID_EMPLOYEE, idEmployee);
+                session.commit();
+            }
+        } catch (Exception e) {
+        } finally {
+            session.close();
+        }
+        return employee;
+    }
+
+    public Employee getHistoryVacation(int idEmployee) {
```

Comment:
Just as a suggestion... this name implies to me that you will return a List or an entity of **HistoryVacation**, but in fact an **Employee** is returned.  So maybe you should return the HistoricVacations only or you can change the method name to be consistent with the operation that the method performs.